### ISSUE TITLE
can not load pretrained model with the deploy prototxt

### ISSUE THREAD

**Issue by ouceduxzk** [shicai/MobileNet-Caffe#28]
Check failed: target_blobs.size() == source_layer.blobs_size() (5 vs. 3) Incompatible number of blobs for layer conv1/bn, this error indicates that i have different blobs in the conv1/bn layer, any idea why this happens?

**Comment by shicai** [comment]
are you using NVIDIA caffe?

**Comment by ouceduxzk** [comment]
No, I am using the latest caffe

**Comment by shicai** [comment]
there are only 3 blobs for batch norm layers. please see: https://github.com/BVLC/caffe/blob/master/src/caffe/layers/batch_norm_layer.cpp

**Comment by ouceduxzk** [comment]
yes, that's what I would expect, I am trying on some other machine , wired error

**Comment by shicai** [comment]
please check the path of your caffe, and the codes of BN layers.

**Comment by ouceduxzk** [comment]
thanks, the problem is solved after checking the path of caffe that is somehow not my caffe even though the path is set in bashrc
